dev/background-api.py

```diff
@@ -0,0 +1,122 @@
+def simulate_toy_bg(alt, az, time):
+    """
+    Function to create dummy bg simulations for test purposes at a particular alt-az pair.
+    Optional: give number of events, instead of observation time.
+    TODO: is there already an established data format?
+
+    Parameters
+    ----------
+    alt : `~astropy.coordinates.Angle`
+        Value of the altitude angle for the simulated observations.
+    az: `~astropy.coordinates.Angle`
+        Value of the azimuth angle for the simulated observations.
+    time : `~astropy.coordinates`
+        Value of the simulated observation time.
+
+    Returns
+    -------
+    image : `~astropy.io.fits.ImageHDU`
+        Image filled with simulated bg events.
+
+    """
+
+
+def simulate_toy_bg(time):
+    """
+    Function to create dummy bg simulations for test purposes.
+    Perform several simulations at different alt-az values, caling recursively simulate_toy_bg(alt, az, time).
+    Optional: give number of events, instead of observation time.
+    TODO: is there already an established data format?
+
+    Parameters
+    ----------
+    time : `~astropy.coordinates`
+        Value of the simulated observation time.
+
+    Returns
+    -------
+    images : array of `~astropy.io.fits.ImageHDU`
+        Array of images filled with simulated bg events. One image per alt-az pair.
+
+    """
+
+
+def group_runs(runlist):
+    """
+    Function that takes a list of runs and groups them in bins of alt-az. The output of the function is one file per alt-az bin with the list of the runs of the corresponding bin. It can be used for both, ON and OFF runlists.
```

Comment:
Can you please wrap lines to ~ 80 characters?
(Such very long lines are difficult to read on Github ... one has to scroll back and forth horizontally.)